src/graphql/query/checkout/initializeCheckout.graphql

```diff
@@ -30,6 +30,10 @@ query initializeCheckout($basketId: String) {
 			value
 			description
 		}
+		upsellsEnabled: uiExperimentSetting(key: "upsells_checkout") {
```

Comment:
@egoroza Please move this Experiment setting to https://github.com/kiva/ui/blob/main/src/graphql/query/checkout/checkoutSettings.graphql and things will start to work.

The `initializeCheckout` is running a network only call and bypassing the cache. Experiment assignment depends on the setting existing in the cache.

Reply:
@egoroza Please remove this additional query block from this file.